scripts/travis/setup-mistral.sh

```diff
@@ -6,8 +6,39 @@ if [ "$(whoami)" != 'root' ]; then
     exit 2
 fi
 
-MISTRAL_STABLE_BRANCH="st2-0.9.0"
+function version_ge() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)" == "$1"; }
+
+# Determine which mistral version to use
+if [[ "${TRAVIS_BRANCH}" == 'master' ]]; then
```

Comment:
Realistically speaking - there is not much we can do right now.

Ideally, we could make parts of st2_deploy.sh more re-usable (so we could re-use parts we need here), but I don't think we want to do this right now.